src/tests/changelist.spec.ts

```diff
@@ -60,10 +60,27 @@ describe('changelist', () => {
     expect(document.querySelector('table > thead')).toBeFalsy()
   })
 
-  it('renders a page when searchFields is defined', async () => {
+  it('renders a search box when searchFields is defined', async () => {
+    const server = app.getHttpServer()
+    const res = await request(server).get(`/admin/user/user`)
+
+    expect(res.status).toBe(200)
+
+    document.documentElement.innerHTML = res.text
+    expect(
+      document.querySelector('body > div.admin-content > form > div > input[name="searchString"]'),
```

Comment:
This selector is too specific, it's going to break as soon as we change the DOM a bit. Just look for a form that contains the input: `form input[name="search"]`